admin/src/views/Stats.vue

```diff
@@ -437,6 +473,43 @@ export default Vue.extend({
   //   },
   // },
   methods: {
+    /** Create chart-digestable data with the option of limiting categories */
+    formatChartData(model: string[], { sort = false, limit = Infinity } = {}): { categories: any[], data: any[] } {
```

Comment:
Planning on removing https://github.com/deltahacks/my.Deltahacks/blob/master/admin/src/views/Stats.vue#L761 in favour of this, since this is more performant